pyanaconda/network.py

```diff
@@ -686,7 +686,7 @@ def ksdata_from_ifcfg(devname, uuid=None):
     if devname in nm.nm_devices():
         # virtual devices (bond, vlan, ...) not activated in installer
         # are not created so guard these checks
-        if nm.nm_device_is_slave(devname):
+        if nm.nm_device_is_slave(devname) and nm.nm_device_type_is_ethernet(devname):
```

Comment:
Yeah I think it the comment would make more sense if moved above the guard. And maybe we need another comment for the added check?